src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,20 @@
 import React from 'react';
+import { Comment } from '../../types/Comment';
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+interface CommentListProps {
+  comments: Comment[] | null;
```

Comment:
Just don't render this component if you don't have comments at all 
```suggestion
  comments: Comment[];
```